src/services/bill-the-lizard/index.js

```diff
@@ -78,14 +94,21 @@ class BillTheLizard {
 		utils.logger(logGroup, 'calling service', host, endpoint, queryParameters);
 
 		return httpRequest(host, endpoint, queryParameters, timeout)
-			.then(response => this.parsePredictions(response));
+			.then(response => overridePredictions(response))
```

Comment:
eslint arrow-parens

Reply:
eslint is configured in the opposite way right now (this change will be rejected by linter)